src/Apps/Artist/Routes/RelatedArtists/__tests__/RelatedArtists.test.tsx

```diff
@@ -0,0 +1,57 @@
+import { RelatedArtistsFixture } from "Apps/__test__/Fixtures/Artist/Routes/RelatedArtistsFixture"
+import { RelatedArtistsRouteFragmentContainer as RelatedArtists } from "Apps/Artist/Routes/RelatedArtists"
+import { MockBoot, renderRelayTree } from "DevTools"
+import { ReactWrapper } from "enzyme"
+import React from "react"
+import { graphql } from "react-relay"
+import { Breakpoint } from "Utils/Responsive"
+
+jest.unmock("react-relay")
+
+describe("RelatedArtists Route", () => {
+  let wrapper: ReactWrapper
+
+  const getWrapper = async (breakpoint: Breakpoint = "xl") => {
+    return await renderRelayTree({
+      Component: RelatedArtists,
+      query: graphql`
+        query RelatedArtists_Test_Query($artistID: String!) {
+          viewer {
+            ...RelatedArtists_viewer
+          }
+        }
+      `,
+      mockResolvers: {
+        Viewer: () => ({
```

Comment:
I got nabbed bad in the Shows one last night and was super confusing... def wouldn't have been able to figure it out had you not posted that comment in your other PR. We should follow up on this because there was nothing I could do to get my connection data to resolve properly. If there was an error or warning or something indicating a Hello World state then it would have been much faster.

Reply:
The docs don’t do a great job either https://www.apollographql.com/docs/graphql-tools/mocking.html#Customizing-mocks.

In short, you can technically stub the whole tree of data starting at a resolver in the root `Query` type. At any place in the tree you may be missing data, graphql-tools will look for that missing type in the list of resolvers. If one of those missing resolvers is one of the few builtin scalar types, then it will fallback to those.

That last part is imo not useful, we should always override them (in createMockNetworkLayer) and raise errors.